back/CinemaReservation.Web/Controllers/AccountController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using CinemaReservation.PresentationLayer.Models;
+using CinemaReservation.BusinessLayer.Models;
+using CinemaReservation.BusinessLayer.Contracts;
+
+namespace CinemaReservation.PresentationLayer.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : Controller
+    {
+        private readonly IAccountService _accountService;
+        public AccountController(IAccountService accountService)
+        {
+            _accountService = accountService;
+        }
+
+
+        [HttpPost("register")]
+        public async Task<IActionResult> PostUserAccount(RegistrationRequest registrationRequest)
+        {
+            if (registrationRequest == null)
+            {
+                return StatusCode(400);
+            }
+
+            RegistrationModel registrationModel = new RegistrationModel(
+                registrationRequest.Name,
+                registrationRequest.Surname,
+                registrationRequest.Email,
+                registrationRequest.Password
+            );
+
+            var response = await _accountService.RegisterUser(registrationModel);
+
+            if (response != null)
+            {
+                return Ok(new AuthorizationResponse(
+                    response.Id,
+                    response.Name,
+                    response.Surname,
+                    response.Email,
+                    response.IsAdmin
+                ));
+            }
+
+            return StatusCode(400);
```

Comment:
When you return some error status codes it is worth to return also message what went wrong with some hints